src/modules/10-common/RouteDefinitions.ts

```diff
@@ -1948,6 +1948,13 @@ const routes = {
     ({ orgIdentifier, projectIdentifier, issueId }: ProjectPathProps & { issueId: string }) =>
       `/sto/orgs/${orgIdentifier}/projects/${projectIdentifier}/ticket-summary/${issueId}`
   ),
+  toSTOOrganizationTicketSettings: withAccountId(
+    ({ orgIdentifier }: OrgPathProps) => `/settings/organizations/${orgIdentifier}/setup/tickets`
+  ),
```

Comment:
```suggestion
  toOrganizationTicketSettings: withAccountId(
    ({ orgIdentifier }: OrgPathProps) => `/settings/organizations/${orgIdentifier}/setup/tickets`
  ),
```
Also, looks like this route should be grouped with similar settings, elsewhere

Reply:
The grouping definitely looks out of place. Let me find somewhere better.